### DISCUSSION TITLE
I have a question

### DISCUSSION THREAD

**Discussion by hjlhym** [ORelio/Minecraft-Console-Client#2690] [General]
Hello everyone，I'm not very good at technology, how do I implement the periodic left button?

**Comment by milutinke** [comment]
> Hello everyone，I'm not very good at technology, how do I implement the periodic left button?

The mcc cannot send a left button arbitrarily, it can attack entities and use right click.
What use case do you have?

**Reply by hjlhym** [reply]
I need to stand on a fixed point to mine blocks on a server to get gold, and the mined manuscript can be destroyed instantly, but the interval between MCC's automatic mining is a bit long
